package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "libp2p-switch",
-  "version": "0.37.3",
+  "version": "0.37.4",
```

Comment:
ah crap. Please do not update the version manually, our release process does it.

Reply:
will fix in master

Reply:
Oh Okay. Sorry. I'm a noob.

Reply:
no worries at all. Shame on me for not reviewing it properly :)